cypress/support/pages/course/pageEditCourse.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+class Edit_Course {
+  static #courseToEdit = '[aria-label="Kurs Mathe"]'
+  static #pageTitle = '[id="page-title"]'
+  static #dropDownCourse = '[class="btn btn-sm dropdown-toggle btn-course-dropdown"]'
+  static #btnCourseEdit = '[class="dropdown-item btn-course-edit"]'
+  static #selectClassesInput = '#classId_chosen > ul > li > input'
+  static #selectClassesDropdown = '#classId_chosen > div > ul > li:nth-child(1)'
+  static #selectClassesDeleteButtons = '#classId_chosen > ul > li > a'
+  static #selectClasses = '#classId_chosen > ul > li > span'
+  static #btnSubmit = '#main-content > section > form > div.modal-footer > button.btn.btn-primary.btn-submit'
+  static #titleEditPageString = 'Kurs bearbeiten'
+  static #classSearchString = 'ALLE'
+  static #numberStudentsInClass = 3
+  static #courseName = 'Mathe'
+  static #selectStudents = '#studentsId_chosen > ul > li'
+
+  clickOnCourse () {
+    cy.get(Edit_Course.#courseToEdit).click()
+  }
+
+  coursePageIsVisible () {
+    cy.url().should('include', '/courses')
+    cy.get(Edit_Course.#pageTitle).should('contain',Edit_Course.#courseName)
```

Comment:
I don't think you need to put a variable here. You can also put the text in here, that you are looking for.
cy.get(Edit_Course.#pageTitle)
cy.contains('Mathe')

Reply:
If we get a pageCourseCommon or so what we should move it there, I think.
Then we need a variable. So I'll wait for an answer before I change it ;-)

Reply:
As d3n153 created a pageCommon.js in the branch added new school administration, these changes should be done after merging both branches.